kubernetes-tests/src/test/java/io/fabric8/crd/generator/zookeeper/v1alpha1/Zookeeper.java

```diff
@@ -24,7 +24,6 @@
 @Version(value="v1alpha1", storage=false)
 @Group("io.zookeeper")
 public class Zookeeper extends CustomResource<ZookeeperSpec, ZookeeperStatus> implements Namespaced {
-  private ZookeeperSpec spec;
```

Comment:
is this a pattern that you need / want to support?
In the e2e tests this was throwing the cycle exception (e.g. the fields are parsed twice if they are explicitly declared in this way).